src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_params.py

```diff
@@ -204,6 +201,11 @@ def load_arguments(self, _):
                    options_list=['--edition'],
                    help='The edition of the database.')
 
+        c.argument('zone_redundant',
+                   options_list=['--zone-redundant', '-z'],
+                   help='The zone redundancy to associate with the database.',
```

Comment:
This description sounds a bit weird to me. How about something like "Specifies whether to enable zone redundancy for the database." And then please add examples to help.py like this:

```
- Create database with zone redundancy enabled
- az sql db create -g ... -s ... -n ... -z

- Create database with zone redundancy explicitly disabled
- az sql db create -g ... -s ... -n ... -z false
```